jetbrains-core/src/software/aws/toolkits/jetbrains/core/tools/ToolPathSelector.kt

```diff
@@ -1,78 +0,0 @@
-// Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
-// SPDX-License-Identifier: Apache-2.0
-
-package software.aws.toolkits.jetbrains.core.tools
-
-import com.intellij.openapi.fileChooser.FileChooserDescriptorFactory
-import com.intellij.openapi.ui.DialogWrapper
-import com.intellij.openapi.ui.TextFieldWithBrowseButton
-import com.intellij.ui.components.JBLabel
-import com.intellij.ui.components.JBTextField
-import com.intellij.ui.components.panels.Wrapper
-import com.intellij.util.text.nullize
-import com.intellij.util.ui.components.BorderLayoutPanel
-import software.aws.toolkits.resources.message
-import java.awt.event.ActionListener
-import java.nio.file.Path
-import java.nio.file.Paths
-import javax.swing.JButton
-
-open class ToolPathSelector<T : ToolType<*>>(private val type: T, private val validationResultHolder: Wrapper) : BorderLayoutPanel() {
```

Comment:
We probably want to keep this since we want to use it in the SAM project wizard, unless we find a better way to configure SAM on that screen

Reply:
yeah i'm looking at SAM now - so might end up pulling this back in

Reply:
In this PR, or another?

Reply:
another